### PR TITLE
Mov 1/setup project

### PR DESCRIPTION
**Pull request by StasAntonov**

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -1,6 +1,12 @@
 plugins {
     id("com.android.application")
     id("org.jetbrains.kotlin.android")
+    id("kotlin-kapt")
```
Comment (IhorPashkovskyi) [inline app/build.gradle.kts]: do we need here kapt we use ksp in project?

File: app/build.gradle.kts [hunk 2]
```diff
@@ -33,6 +42,10 @@ android {
     kotlinOptions {
         jvmTarget = "17"
     }
+    buildFeatures {
+        dataBinding = true
+        buildConfig = true
```
Comment (IhorPashkovskyi) [inline app/build.gradle.kts:46]: do we need this line of code?

File: app/build.gradle.kts (hunk 1 above)
Comment (StasAntonov) [reply app/build.gradle.kts]: fixed

File: app/build.gradle.kts (hunk 2 above)
Comment (StasAntonov) [reply app/build.gradle.kts:46]: needed to generate BilldConfig

**Pull request merged**